mods/default/trees.lua

```diff
@@ -550,7 +560,8 @@ function default.sapling_on_place(itemstack, placer, pointed_thing,
 			interval) then
 		minetest.record_protection_violation(pos, player_name)
 		-- Print extra information to explain
-		minetest.chat_send_player(player_name, "Tree will intersect protection")
+		minetest.chat_send_player(player_name, "Will intersect protection " ..
```

Comment:
`itemstack:get_definition().description` would be helpful here.

Reply:
Done.